bin/main

```diff
@@ -1,104 +1,72 @@
-#!/usr/bin/env ruby
```

Comment:
- [ ] You need to leave the command of the image below to execute.

![image](https://user-images.githubusercontent.com/61479648/93138862-1e3cc000-f6b6-11ea-85e3-63904e1c1c34.png)